x/distribution/client/cli/tx.go

```diff
@@ -127,8 +156,9 @@ $ <appcli> tx set-withdraw-addr cosmos1gghjut3ccd8ay0zduzj64hwre2fxs9ld75ru9p --
 			}
 
 			msg := types.NewMsgSetWithdrawAddress(delAddr, withdrawAddr)
-			return utils.GenerateOrBroadcastMsgs(cliCtx, txBldr, []sdk.Msg{msg})
+			return splitGenerateOrBroadcast(cliCtx, txBldr, []sdk.Msg{msg})
```

Comment:
This is a single message, why split?

Reply:
In my opinion, if splitting is going to be part of every command in the short term (you need to decide on this..) I might even move the functionality inside `utils.GenerateOrBroadcastMsgs` and accessible via context.

I can remove from here but for the sake of consistency, maybe the best option is that we move it inside GenerateOrBroadcastMsgs and irrespective of the command, the user can always have the option to split.

Reply:
We haven't decided on this atm -- it should only be on the withdraw rewards command.

Reply:
ok, I can change that. No problem.
Still, it is something that might result useful in the future as other projects that use the SDK may find that they also have a significant amount of messages in a single tx and may need to split.
I would recommend considering having this feature in all commands.
I will rework this and ping you when it is ready.